packs/archivista-0.8.0/README.md

```diff
@@ -0,0 +1,53 @@
+# Archivista
+
+![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+
+Helm chart for Archivista - a graph and storage service for in-toto attestations.
+
+## Prerequisites
+
+A MySQL database and S3 compatible store are needed to successfully install this Helm chart.
+Refer to the [Archivista configuration](https://github.com/in-toto/archivista#configuration) configuration guide for environment variables needed
+to establish connections to each datastore. These environment variables can be added to this Helm chart using the value `deployment.env[]`.
+
+## Parameters
+
+| Key                                        | Type   | Default       |
+|---                                         |---     |---            |
+| affinity                                   | object | `{}`          |
+| autoscaling.enabled                        | bool   | `false`       |
+| autoscaling.maxReplicas                    | int    | `10`          |
+| autoscaling.minReplicas                    | int    | `1`           |
+| autoscaling.targetCPUUtilizationPercentage | int    | `80`          |
+| deployment.env                             | list   | `[]`          |
+| fullnameOverride                           | string | `""`          |
+| image.pullPolicy                           | string | `"IfNotPresent"` |
+| image.repository                           | string | `"ghcr.io/testifysec/archivista"` |
+| image.tag                                  | string | `"0.1.1"`     |
+| ingress.annotations                        | object | `{}`          |
+| ingress.className                          | string | `""`          |
+| ingress.enabled                            | bool   | `true`        |
+| ingress.hosts[0].host                      | string | `"archivista.localhost"` |
+| ingress.hosts[0].path                      | string | `"/"`         |
+| ingress.tls                                | list   | `[]`          |
+| nameOverride                               | string | `""`          |
+| nodeSelector                               | object | `{}`          |
+| podAnnotations                             | object | `{}`          |
+| podSecurityContext                         | object | `{}`          |
+| replicaCount                               | int    | `1`           |
+| resources                                  | object | `{}`          |
+| serviceAccount.annotations                 | object | `{}`          |
+| serviceAccount.create                      | bool   | `false`       |
+| serviceAccount.name                        | string | `""`          |
+| service.port                               | int    | `8082`        |
+| service.type                               | string | `"ClusterIP"` |
+| tolerations                                | list   | `[]`          |
+
+## Usage
+Please refer to [Archivista Github](https://github.com/in-toto/archivista/blob/main/README.md) for further information.
```

Comment:
```suggestion
You can find additional guidance in the [Archivista Github](https://github.com/in-toto/archivista/blob/main/README.md) README.
```

Super tiny suggestion :)

Reply:
I'd prefer if we left it out, I'll encourage InToto and the Archivista people to update the Readme as they see fit.